app/assets/javascripts/appointments.js.coffee

```diff
@@ -240,7 +265,7 @@ $ ->
   $(document).on 'change', '#appointment_content_indications', ->
     appointment_id = $(this).parents('.row.appointment').data('id')
     contents = $(this).val()
-    data = 'contents' : contents
+    data = appointment: 'contents' : contents
```

Comment:
Is the key a sting for a reason? Should it be: data = appointment: contents: contents